### PR TITLE
Removed extra services and simplified Durpal settings.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**

### CODE REVIEW

File: .docker/settings.php [hunk 1]
```diff
@@ -2,90 +2,36 @@
...
-// Environment specific settings files.
-if (getenv('LAGOON_ENVIRONMENT_TYPE')) {
-  if (file_exists(__DIR__ . '/' . getenv('LAGOON_ENVIRONMENT_TYPE') . '.settings.php')) {
-    include __DIR__ . '/' . getenv('LAGOON_ENVIRONMENT_TYPE') . '.settings.php';
-  }
-}
-
 // Last: this servers specific settings files.
 if (file_exists(__DIR__ . '/settings.local.php')) {
   include __DIR__ . '/settings.local.php';
```
Comment (thom8) [inline .docker/settings.php]: This block isn't required

File: .docker/settings.php [hunk 2]
```diff
@@ -2,90 +2,36 @@
...
+  'driver' => 'mysql',
+  'database' => getenv('MARIADB_DATABASE') ?: 'drupal',
+  'username' => getenv('MARIADB_USERNAME') ?: 'drupal',
+  'password' => getenv('MARIADB_PASSWORD') ?: 'drupal',
+  'host' => getenv('MARIADB_HOST') ?: 'mariadb',
+  'port' => 3306,
+  'prefix' => '',
+];
+
+$conf['search_api_override_mode'] = 'load';
```
Comment (thom8) [inline .docker/settings.php]: This is D8 solr config, it can go, you can still create solr servers manually.

File: docker-compose.yml [hunk 3]
```diff
@@ -46,7 +46,7 @@ services:
     << : *default-volumes
     environment:
       << : *default-environment
-      LAGOON_LOCALDEV_URL: ${LOCALDEV_URL_NGINX:-http://govcms-nginx.docker.amazee.io}
+      LAGOON_LOCALDEV_URL: ${LOCALDEV_URL:-http://govcms.docker.amazee.io}
```
Comment (AlexSkrypnyk) [inline docker-compose.yml]: @thom8 not sure about this now. There are 2 variable for this. which one to use?

**Review by thom8 — COMMENTED** [review]
Looks good just remove everything we can from settings.php

**Review by thom8 — APPROVED** [review]
Nice, much cleaner :)

File: docker-compose.yml (hunk 3 above)
Comment (thom8) [reply docker-compose.yml]: pls use `LOCALDEV_URL` and drop `LOCALDEV_URL_NGINX` also in `.env.default`

**Review by thom8 — COMMENTED** [review]
[ignore]

**Review by thom8 — APPROVED** [review]

**Pull request merged**